easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-2022.02.0-443-rhel-x86_64.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'Binary'
+
+name = 'RStudio-Server'
+version = '2022.02.0-443'
+versionsuffix = '-rhel-%s' % ARCH
+
+homepage = 'https://www.rstudio.com/'
+description = """This is the RStudio Server version.
+RStudio is a set of integrated tools designed to help you be more productive with R.
+
+The server can be started with:
+  rserver --server-daemonize=0 --www-port 8787 --rsession-which-r=$(which R)
+"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://download2.rstudio.org/server/rhel8/%s' % ARCH]
+sources = ['rstudio-server-rhel-%%(version)s-%s.rpm' % ARCH]
+checksums = ['8eb698ddde2043b9f143296c46a4b1643e7a4b5373740f15e016bf2da1ad6289']
+
+# both rpm2cio (provided by rpm package) and cpio commands are required for installing
+osdependencies = ['cpio', 'rpm']
+
+install_cmd = "rpm2cpio rstudio-server*.rpm | cpio -idmv && cp -a usr/lib/rstudio-server/* %(installdir)s"
+
+sanity_check_paths = {
+    'files': ['bin/rstudio-server'],
+    'dirs': ['db', 'R', 'resources'],
```

Comment:
Just wondering: does the `R` here mean that this binary installation comes with it's own `R` installation as well? Since I see `R` in the list of dependencies for those `RStudio-Server` EasyConfigs that build from sources, I was already wondering how this one would ensure that an `R` installation is present. But if it's shipped with the `rpm` for `RStudio-Server`, then that would explain :)

Reply:
Also: just to check, are the other dirs listed [here](https://github.com/easybuilders/easybuild-easyconfigs/blob/ceded760bf1db1483df14c5eb93a6ac00fd8846c/easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-1.4.1717-foss-2021a-Java-11-R-4.1.0.eb#L70) not present for the binary installation?

Reply:
no, there is no R in the package. it's intentional. we only need to load the webservice, loading R is mandatory for actual usage, but not for the webservice.

Reply:
Sorry for slightly off-topic; i've actually think that we should do the same for our source builds; dropping R as a dep and moving RStudio-Server up to GCCcore, allowing (and requiring) users to select either R from either toolchain. Maybe less of an issue now that fosscuda has gone away, and noone seems to care about intel compiled R recently.